Lib/fontTools/subset.py

```diff
@@ -1254,6 +1254,23 @@ def collect_features(self):
     return _uniq_sort(sum((s.Script.collect_features()
                            for s in self.ScriptRecord), []))
 
+# CBLC will inherit it
+@_add_method(ttLib.getTableClass('EBLC'))
+def subset_glyphs(self, s):
+  for strike in self.strikes:
+    for indexSubTable in strike.indexSubTables:
+      indexSubTable.names = [n for n in indexSubTable.names if n not in s.glyphs]
+    strike.indexSubTables = [i for i in strike.indexSubTables if i.names]
+  self.strikes = [s for s in self.strikes if s.indexSubTables]
+
+  return True
+
+# CBDC will inherit it
+@_add_method(ttLib.getTableClass('EBDT'))
+def subset_glyphs(self, s):
+  self.strikeData = [_dict_subset(strike, s.glyphs) for strike in self.strikeData]
```

Comment:
this raises `KeyError` whenever a glyph name is in the list of glyphs to retain `s.glyphs` but is not also among the keys of the `strike` dictionary.

For example, in the example color fonts from:
https://github.com/googlei18n/color-emoji/tree/master/specification/examples

the glyph '.notdef' is not contained in the CBDT strikeData, but is contained in the post table, so the `_dict_subset` function fails with KeyError.

I think you should subset the CBDT table's strikeData like that:

```python
self.strikeData = [{g: strike[g] for g in s.glyphs if g in strike}
                   for strike in self.strikeData]
```